chain-impl-mockchain/src/testing/gen/vote.rs

```diff
@@ -83,6 +86,32 @@ impl VoteTestGen {
         )
     }
 
+    pub fn committee_members_manager(
+        members_no: usize,
+        threshold: usize,
+    ) -> CommitteeMembersManager {
+        let crs_seed = b"This should be a shared seed among the different committee members. Could be the id of the previous VotePlan";
+        let mut rng = ChaCha20Rng::from_seed([0u8; 32]);
+        CommitteeMembersManager::new(&mut rng, crs_seed, threshold, members_no)
+    }
+
+    pub fn private_vote_plan() -> VotePlan {
+        Self::private_vote_plan_with_committees_manager(&Self::committee_members_manager(3, 1))
+    }
+
+    pub fn private_vote_plan_with_committees_manager(
+        manager: &CommitteeMembersManager,
+    ) -> VotePlan {
```

Comment:
Now this could be more elegant a method `with_committee_manager` modifying `self` in a builder pattern, but the rest of the impl of this type defines `self`-less methods. The `VoteTestGen` type currently has no purpose, for namespacing a module is more typically used in Rust.

Reply:
can i propose to change it in another pr as VoteTestGen is spread around several crate?